frontend/jest/playwright.setup.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * `frontend/jest-playwright.config.js` is for configuring Playwright's launch config options
+ * `frontend/e2e/playwright.setup.js` is for configuring `jest`, `browser`,
+ * and `page` objects
+ */
+
+// (thuang): This is the max time a test can take to run.
+// Since when debugging, we run and !headless, this means
+// a test can take more time to finish, so we don't want
+// jest to shut off the test too soon
+jest.setTimeout(2 * 60 * 1000);
+
+// (thuang): Please make sure this number matches
+// `RETRY_ATTEMPTS` in `jest/screenshot_env.js` (when we add screenshot)
+jest.retryTimes(2);
+
+beforeEach(async () => {
+  const client = await page.context().newCDPSession(page);
+
+  await client.send("Animation.setPlaybackRate", {
+    playbackRate: 12,
```

Comment:
Set playback rate to the fastest to "disable" animation related flakiness

Reply:
can you add this as a comment. Otherwise it's just a magic number.